CIPs/0022.md

```diff
@@ -0,0 +1,146 @@
+# CIP [0022]: Upgrade epoch SNARK data for flexibility, slashability and security
+
+---
+cip: 22
+title: Upgrade epoch SNARK data for flexibility, slashability and security
+author: Kobi Gurkan <me@kobi.one>
+discussions-to: <URL>
```

Comment:
Can you add a discussions-to url? It can be as easy as opening a Github ticket that is called: "Discussions to  Upgrade epoch SNARK data for flexibility, slashability and security" and linking the URL here. Or, if you prefer, you can make one on the forum.celo.org.